src/main/java/duke/duke/Duke.java

```diff
@@ -0,0 +1,178 @@
+package duke;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+    /**
+     * main class for Duke todolist
```

Comment:
Calling it todolist might be misleading as it contains events, deadlines and todos. May want to consider updating the name.